### ISSUE TITLE
txt file for the KITTI submission 

### ISSUE THREAD

**Issue by ganyz** [2gunsu/monocon-pytorch#5]
Hello author, thanks for the great contribution!

I want to ask **the Question**: 
(1) when i finish training the trainval data,
     how to create the object results file (.txt) to submit to the KITTI.
     I can't find it from the test.py >.<
(2) have you trained the trainval results to submit to the KITTI test set benchmark?
     Can the paper's test result be reproduce?

Thank you!

**Comment by 2gunsu** [comment]
Hello. Thanks for your interest in my code. 😊

The function to generate the submission file (.txt file) you mentioned is in need of implementation. We plan to implement it as soon as possible, but my personal research is urgent and the implementation is being delayed. 😢

Once implemented, I will add a comment to this issue again. Really sorry!

**Comment by ganyz** [comment]
> Hello. Thanks for your interest in my code. 😊
> 
> The function to generate the submission file (.txt file) you mentioned is in need of implementation. We plan to implement it as soon as possible, but my personal research is urgent and the implementation is being delayed. 😢
> 
> Once implemented, I will add a comment to this issue again. Really sorry!

Yr a nice man ! Best wishes to you !

**Comment by 2gunsu** [comment]
Thank you~
If there are any updates, I will leave a comment here again.

Have a nice day!😊
